packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -314,16 +314,33 @@ function useFocusContainment(scopeRef: RefObject<Element[]>, contain: boolean) {
 
       let focusedElement = document.activeElement;
       let scope = scopeRef.current;
-      if (!isElementInScope(focusedElement, scope)) {
+      let childScopeRef = isElementInChildScope(focusedElement, scopeRef);
```

Comment:
Perhaps a bit of a nitpick, but  the naming/behavior `isElementInChildScope` is a bit confusing now that it returns a boolean or an actual scopeRef (which only happens when portals are involved). Could we rename it or split it into separate functions or have it just return a object that has `{contains: bool, containingScope: scopeRef}` or something along those lines

Reply:
yeah, it's only used internally, so i can do that